sdk/core/azure-core/CHANGELOG.md

```diff
@@ -222,11 +235,21 @@
 - Fixed setting up the default transport adapter.
 - Fixed linker error of missing pthread on Linux.
 - Initialize class data members to avoid MSVC warning.
-- Throw `Azure::Core::Http::TransportException` if `curl_easy_init()` returns a null handle.
+- Throw `Azure::Core::Http::TransportException` if `curl_easy_init()` returns a null handle. (A community contribution, courtesy of _[ku-sourav](https://github.com/ku-sourav)_)
 
 ### Other Changes and Improvements
 
 - Added support for distributing the C++ SDK as a source package via vcpkg.
+- Fixed installation error when the SDK is being installed under a non-standard prefix. (A community contribution, courtesy of _[juchem](https://github.com/juchem)_)
+- Fixed linker error when pthread is missing on Linux. (A community contribution, courtesy of _[lordgamez](https://github.com/lordgamez)_)
+
+### Acknowledgments
+
+Thank you to our developer community members who helped to make Azure Core better with their contributions to this release:
+
+- Gabor Gyimesi _([GitHub](https://github.com/lordgamez))_
+- Marcelo Juchem _([GitHub](https://github.com/juchem))_
+- `ku-sourav` _([GitHub](https://github.com/ku-sourav))_
```

Comment:
This is fine, but I am curious, why did you decide to format "non-names" with back ticks?

Reply:
To distinguish between GitHub profile name (the best info that we have; but it is not a pseudonym - the name in profile is simply missing), and the actual name/alias/pseudonym.

@ku-sourav, please let me know if you want to be mentioned here differently (your name).